doc/manual.html

```diff
@@ -2066,8 +2069,8 @@ <h3 id="api_misc">Miscellaneous methods</h3>
       instance, you can call this method with a function argument. It
       will call the function, buffering up all changes, and only doing
       the expensive update after the function returns. This can be a
-      lot faster. The return value from this method will be the return
-      value of your function.</dd>
+      lot faster. This method return value, returned by your
+      function.</dd>
```

Comment:
"This method return value, returned by your function." isn't a grammatically correct sentence.